scanners/git-repo-scanner/templates/git-repo-scanner-scan-type.yaml

```diff
@@ -24,7 +24,7 @@ spec:
           restartPolicy: OnFailure
           containers:
             - name: git-repo-scanner
-              image: "{{ .Values.scanner.image.repository }}:{{ .Values.scanner.image.tag | default .Chart.Version }}"
+              image: "{{ .Values.scanner.image.repository }}:{{ .Values.scanner.image.tag | default .Chart.AppVersion }}"
```

Comment:
The `Version` here and in (zap-advanced & test-scan) is used intentionally as these are our own tools / or wrappers we release in our normal versioning scheme. I'm open to look into alternatives but this would require changing this also in our CI builds.